Modules/_datetimemodule.c

```diff
@@ -1336,10 +1346,10 @@ tzinfo_from_isoformat_results(int rv, int tzoffset, int tz_useconds)
 static PyObject *
 format_ctime(PyDateTime_Date *date, int hours, int minutes, int seconds)
 {
-    static const char * const DayNames[] = {
+    static const char * const DayNames[] = {  // Static is okay here (immutable data).
```

Comment:
Are these comments really useful? To me, it seems obvious that static constants are safe. I'm not sure of the value of such comment.

Reply:
A comment sounds useful to flag all local statics as "good" and for a linter that checks for unverified local static vars. I'm ok with the comments, but maybe a shorter one like ``// static ok``.?

Reply:
I agree with @vstinner. If you really want to keep these comments, please move them to separate lines to avoid noise when using `git blame`.